pages/EditPage/[EditPageId].vue

```diff
@@ -56,42 +63,63 @@ type User2 = {
     //PageDonations: PageDonation[]
     //DonationPayouts: DonationPayout[]  
 }
+const data_family = ref<Family>({
+    cuid: "",
+    Stripe_Account_id: "",
+    Stripe_Accont_cuid: "",
+    created_at: "",
+    updated_at: Date.toString(),
+    family_name: "",
+    advocateCuid: cvuser2.value.cuid 
+
+})
+
+const isAdvocate = computed(() => cvuser.value?.user_role == "advocate")
+const data_all_users = ref<Family[]>([])
 
 const imageData = ref<Image[]>([])
```

Comment:
need to add family select - if the current user is a family member, can automatically assume the page is for that family and dont show the dropdown